applications/CompressiblePotentialFlowApplication/python_scripts/apply_far_field_process.py

```diff
@@ -18,61 +23,66 @@ def __init__(self, Model, settings ):
                 "mesh_id": 0,
                 "inlet_phi": 1.0,
                 "velocity_infinity": [1.0,0.0,0]
-            }  """ );
+            }  """ )
 
 
         settings.ValidateAndAssignDefaults(default_parameters);
 
-        self.model_part = Model[settings["model_part_name"].GetString()]
+        self.far_field_model_part = Model[settings["model_part_name"].GetString()]
         self.velocity_infinity = KratosMultiphysics.Vector(3)#array('d', [1.0, 2.0, 3.14])#np.array([0,0,0])#np.zeros(3)#vector(3)
         self.velocity_infinity[0] = settings["velocity_infinity"][0].GetDouble()
         self.velocity_infinity[1] = settings["velocity_infinity"][1].GetDouble()
         self.velocity_infinity[2] = settings["velocity_infinity"][2].GetDouble()
         #self.density_infinity = settings["density_infinity"].GetDouble() #TODO: must read this from the properties
-        self.inlet_phi = settings["inlet_phi"].GetDouble()
-        self.model_part.ProcessInfo.SetValue(CompressiblePotentialFlowApplication.VELOCITY_INFINITY,self.velocity_infinity)
+        self.inlet_phi_0 = settings["inlet_phi"].GetDouble()
+        self.far_field_model_part.ProcessInfo.SetValue(CPFApp.VELOCITY_INFINITY,self.velocity_infinity)
 
 
 
     def Execute(self):
-        #KratosMultiphysics.VariableUtils().SetVectorVar(CompressiblePotentialFlowApplication.VELOCITY_INFINITY, self.velocity_infinity, self.model_part.Conditions)
-        for cond in self.model_part.Conditions:
-            cond.SetValue(CompressiblePotentialFlowApplication.VELOCITY_INFINITY, self.velocity_infinity)
-
-        #select the first node
-        for node in self.model_part.Nodes:
-            node1 = node
+        #KratosMultiphysics.VariableUtils().SetVectorVar(CPFApp.VELOCITY_INFINITY, self.velocity_infinity, self.far_field_model_part.Conditions)
+        for cond in self.far_field_model_part.Conditions:
+            cond.SetValue(CPFApp.VELOCITY_INFINITY, self.velocity_infinity)
```

Comment:
why does every condition need this?

Reply:
https://github.com/KratosMultiphysics/Kratos/pull/4661#discussion_r275655518